concrete/src/Attribute/Controller.php

```diff
@@ -420,9 +420,19 @@ public function field($fieldName)
         return 'akID[' . $this->attributeKey->getAttributeKeyID() . '][' . $fieldName . ']';
     }
 
+    /**
+     * Get the ID to use for label elements. Not applicable in form views that do
+     * not contain <label>
+     * @return string
+     */
+    public function getControlID()
+    {
+        return $this->field('value');
+    }
+
     /**
      * Get the ID to use for label elements.
-     *
+     * @deprecated
```

Comment:
Since this is deprecated it should be updated to use the non-deprecated logic:

```php
public function getLabelID()
{
    return $this->getControlID();
}
```

Reply:
This will also fix BC issues from the select attribute controller change